net/respondd/README.md

```diff
@@ -1,3 +1,109 @@
+# What is respondd?
+
+## Introduction
+respondd is a server distributing information within a network.
+
+For doing so, respondd spawns a udp socket (1001/udp) joining a multicast group. When a request is received, the information requested is transmitted to the requestor. 
+
+All information is oranized in a non-hierachical namespace. Each entry identifies a *provider* implemented in at least one C module. (eg `statistics`, `nodeinfo`, `...`) - each request queries a specific provider. 
+Each response is the merge-result of the output of all modules registered for the provider.
+
+Some providers are implemented in [/package/gluon-respondd](https://github.com/freifunk-gluon/gluon/tree/master).
+## Usage
+
+TODO: Why is the port number included in the multicast group, too?
+```
+respondd [-p <port>] [-g <group> -i <if0> [-i <if1> ..]] [-d <dir>]
+  -p <int>         port number to listen on
+  -g <ip6>         multicast group, e.g. ff02::2:1001
+  -i <string>      interface on which the group is joined
+  -d <string>      data provider directory (default: current directory)
+  -h               this help
+```
+
+
+
+## Procotol
+
+Request and response are encoded as *utf8*-strings. These strings are sent as UDP-Packets. Fragmentation is not supported -  gzip-compression (for the response) is optional. 
+
+- The request is the name for the provider - '`GET `' can be prepended for compression
+- The response is encoded as JSON. The structure is determined by the actual data.
+
+### Example 
+Requesting `nodeinfo` as implemented in the freifunk-gluon modules.
+
+#### Request
+```
+nodeinfo
+```
+
+#### Response
+```json
+"nodeinfo": {
```

Comment:
This isn't actually valid JSON (and thus probably not copied from an actual output?). You are missing a surrounding pair of `{}`. And as mentioned above, this is actually the (decompressed) output of a `GET nodeinfo` request, a pure request of `nodeinfo` wouldn't have wrapped the return data in another object with the key `nodeinfo`.